index.html

```diff
@@ -3238,7 +3266,7 @@ <h3>Connection Through Proxy Synchronization</h3>
                 </p>
                 <figure id="deployment-service-sync-proxy">
                     <img src="images/deployments/arch-deploy-service-sync-intermediary.png"
-                         class="wot-arch-diagram" />
+                         class="wot-arch-diagram" alt="service sync proxy" />
```

Comment:
might want to call it "intermediary" also in the alt text.

Reply:
In that case, maybe it would be better to use "deployment-service-sync-intermediary" for id as well. What do you think?

Reply:
Yes, sounds good. Need to double check if it is referenced from somewhere and fix there as well...

Reply:
OK. I've checked the index.html file again for "proxy" and converted it to "intermediary" if appropriate, e.g., the PNG file's name is *-intermediary.png".

Let's review the changes during the Architecture call tomorrow on May 23.